tests/browscap-settings-isolation-test.php

```
@@ -0,0 +1,155 @@
+{
+    // Fixed behaviour: only touch browscap_last_modified in the stored copy
+    $runtime_settings['browscap_last_modified'] = $new_timestamp;
+    $stored_settings['browscap_last_modified']  = $new_timestamp;
+    // update_option('slimstat_options', $stored_settings)  ← only stored copy written
+    return [$stored_settings, $runtime_settings];
+}
+
+/**
+ * Reproduce the BUGGY Browscap write logic (for reference / contrast).
+ */
+function browscap_write_buggy(array $runtime_settings, array $stored_settings, int $new_timestamp): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$stored_settings'. (undefined)

(UnusedFormalParameter)